src/app/login/login.component.html

```diff
@@ -5,6 +5,8 @@
     <div class="login-body">
         <div class="login-box">
             <h3>Login</h3>
+            <br>Using your Climate API account information
```

Comment:
I'd recommend leaving it--the information in this case refers to email and password; if you just say your account, people might question if they're logging into the right place.